src/ripple/app/main/Main.cpp

```diff
@@ -588,10 +587,6 @@ int run (int argc, char** argv)
 //
 int main (int argc, char** argv)
 {
-    // Workaround for Boost.Context / Boost.Coroutine
-    // https://svn.boost.org/trac/boost/ticket/10657
-    (void)beast::currentTimeMillis();
```

Comment:
From the comment, the purpose of this code is to call `GetTimeZoneInformation` before the first coroutine is launched. The bug is marked as "wontfix". So don't we still need this call?

Reply:
Do we know what the failure mode would be?  Perhaps @bachase can try building this branch and see if it behaves well for him, since he runs Windows.

Reply:
It looks like it happens on the windows `_f_time` call. The `timeptr` struct contains timezone information: https://msdn.microsoft.com/en-us/library/z54t9z5f.aspx

I'd prefer to leave this in, even if coro's don't currently call `GetTimezoneInformation`. If it's added later it would be a PITA to track down the crash.

Reply:
I'm not observing a crash on windows with this change when running the unit tests. Using the example code on the boost ticket, I also do not observe a crash with VS 2017 on Windows 10. I do not have other versions of windows available to test.  Even though I'm unable to recreate, I'm with @seelabs and would prefer not have this bite us in the future. 

I can try more testing to see if I can recreate, but frankly this seems fairly low priority relative to other work. I'm fine with @scottschurr's suggestion of just moving the code inside the existing windows specific startup block.

Reply:
Thanks @bachase.  I have to agree with your assessment of the priority of this work.  I'll make a stab at a Windows only version of the code.